dev/tools/controllerbuilder/generate.sh

```diff
@@ -108,12 +108,13 @@ go run . generate-types  \
 # BigQueryDataTransferConfig
 go run . generate-types \
     --service google.cloud.bigquery.datatransfer.v1 \
-    --api-version bigquerydatatransfer.cnrm.cloud.google.com/v1alpha1 \
-    --resource BigQueryDataTransferConfig:TransferConfig
+    --api-version bigquerydatatransfer.cnrm.cloud.google.com/v1beta1 \
+    --resource BigQueryDataTransferConfig:TransferConfig \
+    --skip-scaffold-files # skipping because the files were generated using a previous pattern, making them incompatible with the new scaffolding approach.
```

Comment:
Yeah this is kind of needed for re-running the codegen commands. Because we updated both the naming of the scaffolded files and the location of the external reference and identity code.

Reply:
Could you update the direct guide ?

Reply:
Ideally, new resources shouldn't require this flag. It's only intended for legacy direct resources created while the code template was being updated. We want the codegen tool to be able to be rerun without requiring the flag. I'm looking into the possibility of adding a presubmit check to ensure the tool can be rerun on PR.